lib/spaceship/tunes/tunes_client.rb

```diff
@@ -745,8 +756,8 @@ def create_tester!(tester: nil, email: nil, first_name: nil, last_name: nil)
         req.headers['Content-Type'] = 'application/json'
       end
 
-      data = parse_response(r, 'data')['testers']
-      handle_itc_response(data) || data[0]
+      data = parse_response(r, 'data')[arr_key]
+      handle_itc_response(data) || data.detect { |element| element['emailAddress']['value'] == email }
```

Comment:
I see that in other places, we compare emails with `casecmp`. Is this something to do here as well ?